src/reconcile/rec_col.c

```diff
@@ -1058,8 +1058,8 @@ __wt_rec_col_var(
                       (last.start_ts == start_ts && last.start_txn == start_txn &&
                         last.stop_ts == stop_ts && last.stop_txn == stop_txn)) &&
                   ((deleted && last.deleted) ||
-                      (!deleted && !last.deleted && last.value->size != 0 &&
-                        last.value->size == size && memcmp(last.value->data, data, size) == 0))) {
+                      (!deleted && !last.deleted && last.value->size == size &&
```

Comment:
Just as an FYI this last.value->size != 0 check was added as a temporary way of getting clang sanitizer to stop complaining. Hence why I'm removing it again.

Reply:
Thanks. I was just about to ask about that.